components/dialog/test/dialog-fullscreen.vdiff.js

```diff
@@ -0,0 +1,79 @@
+import '../dialog-fullscreen.js';
+import { expect, fixture, html, nextFrame } from '@brightspace-ui/testing';
+import { footer, general, long, tabs } from './dialog-shared-contents.js';
+import { ifDefined } from 'lit/directives/if-defined.js';
+
+function createDialog(opts) {
+	const defaults = { content: html`${general}${footer}`, noPadding: false };
+	const { content, noPadding, width } =  { ...defaults, ...opts };
+	return html`
+		<d2l-dialog-fullscreen title-text="Dialog Title" ?no-padding="${noPadding}" width="${ifDefined(width)}" opened>
+			${content}
+		</d2l-dialog-fullscreen>
+	`;
+}
+
+function dispatchFullscreenWithinEvent(elem, state) {
+	elem.dispatchEvent(new CustomEvent(
+		'd2l-fullscreen-within', { bubbles: true, composed: true, detail: { state: state } }
+	));
+}
+
+describe('dialog-fullscreen', () => {
+
+	[/*'native',*/ 'custom'].forEach((type) => {
+
+		describe(type, () => {
+			before(async() => {
+				window.D2L.DialogMixin.preferNative = type === 'native';
+			});
```

Comment:
Nit: shouldn't need to be async

```suggestion
			before(() => window.D2L.DialogMixin.preferNative = type === 'native');
```